src/test/java/signUpTesting/VerifySignUpLink.java

```diff
@@ -0,0 +1,27 @@
+package signUpTesting;
+
+import io.qameta.allure.*;
+import org.openqa.selenium.By;
+import org.testng.annotations.*;
+import utils.DriverSetUp;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class VerifySignUpLink extends DriverSetUp {
+
+    @Test(description = "Verification of 'Sign up' link presence")
+    @Description ("Test description: check that 'Sign up' link is present")
+    @Step("Check that 'Sign up' link is present")
+    public void verifySignUpLinkPresent() {
+        driver.findElement(By.xpath("//a[text()='Sign up']")).isDisplayed();
+    }
+
+    @Test(description = "Verification of 'Sign up' link title correctness", dependsOnMethods = {"verifySignUpLinkPresent"})
+    @Description("Test description: check that 'Sign up' title is correct")
+    @Step("Check that 'Sign up' link title is correct")
+    public void verifySignUpLinkCorrectTitle() {
+        String signUpLink = driver.findElement(By.xpath("//a[text()='Sign up']")).getText();
```

Comment:
The variable represent signUpLinkText , but not signUpLink... please be more precise while naming. 
Thats why your next assert a bit confused me at first.

Reply:
Variable name is changed to signUpLinkText in commit
30dc54fd9e13c250acf9eb8e72598d0ee2d4d232